rust/parquet/Cargo.toml

```diff
@@ -29,7 +29,7 @@ build = "build.rs"
 edition = "2018"
 
 [dependencies]
-parquet-format = "2.6.1"
+parquet-format = "~2.6.1"
```

Comment:
nit: can we add a comment here for future reference? I think this is not the first time we got this error.

Reply:
@sunchao @alamb @mqy given the long CI times, is it okay if we piggy-back on #9133 to add the comment?

Reply:
Sure. SGTM.

Reply:
yes for sure

Reply:
The update note was added to https://github.com/apache/arrow/pull/9133/commits/8cf807f8ba1e4a2f769dad8a1b1f63dc725eb34e